test_plus/test.py

```diff
@@ -266,7 +266,11 @@ def make_user(cls, username='testuser', password='password', perms=None):
             user_factory = User.objects.create_user
 
         USERNAME_FIELD = getattr(User, 'USERNAME_FIELD', 'username')
-        test_user = user_factory(**{USERNAME_FIELD: username})
+        EMAIL_FIELD = getattr(User, 'EMAIL_FIELD', None)
+        user_data = {USERNAME_FIELD: username}
```

Comment:
Let's switch these two lines to keep the logic grouping together based on the variable names:

```suggestion
        user_data = {USERNAME_FIELD: username}
        EMAIL_FIELD = getattr(User, 'EMAIL_FIELD', None)
```